internal/proxy/meta_cache.go

```diff
@@ -1206,7 +1205,6 @@ func (m *MetaCache) GetDatabaseInfo(ctx context.Context, database string) (*data
 		dbInfo := &databaseInfo{
 			dbID:             resp.GetDbID(),
 			createdTimestamp: resp.GetCreatedTimestamp(),
-			properties:       funcutil.KeyValuePair2Map(resp.GetProperties()),
```

Comment:
Should not cache properties since there is no refresh mechanism to update them after changes.